it/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingMessagesWithWorkItemItSpec.scala

```diff
@@ -232,6 +249,38 @@ class PollingMessagesWithWorkItemItSpec extends PlaySpec
 
     }
 
+    "if finds nothing mark work item as failed so can be retried" in {
+
+      val movementRepository = new MovementRepository(
+        mongoComponent,
+        appConfig,
+        timeService
+      )
+
+      wireMock.resetAll()
```

Comment:
Needed it here because after BeforeEach resets the wiremock it then setups  lrn 1 to return contents. So its either do this or change all the other tests to call the setUpWireMockStubs() and remove it from beforeEach